cmd/tfplugindocs/testdata/scripts/schema-json/migrate/time_provider_idempotency.txtar

```diff
@@ -37,13 +38,25 @@ cmpenv examples/resources/sleep/example_3.tf exp-examples/resources/sleep/exampl
 cmpenv examples/resources/sleep/import_1.sh exp-examples/resources/sleep/import_1.sh
 cmpenv examples/resources/sleep/import_2.sh exp-examples/resources/sleep/import_2.sh
 
-cmpenv examples/resources/static/example_1.tf examples/resources/static/example_1.tf
-cmpenv examples/resources/static/example_2.tf examples/resources/static/example_2.tf
-cmpenv examples/resources/static/import_1.sh examples/resources/static/import_1.sh
+cmpenv examples/resources/static/example_1.tf exp-examples/resources/static/example_1.tf
+cmpenv examples/resources/static/example_2.tf exp-examples/resources/static/example_2.tf
+cmpenv examples/resources/static/import_1.sh exp-examples/resources/static/import_1.sh
+
+cmpenv examples/ephemeral-resources/ephemeral_static/example_1.tf exp-examples/ephemeral-resources/ephemeral_static/example_1.tf
+cmpenv examples/ephemeral-resources/ephemeral_static/example_2.tf exp-examples/ephemeral-resources/ephemeral_static/example_2.tf
+cmpenv examples/ephemeral-resources/ephemeral_static/import_1.sh exp-examples/ephemeral-resources/ephemeral_static/import_1.sh
 
 -- expected-output.txt --
 migrating website from "$WORK/docs" to "$WORK/templates"
-migrating functons directory: functions
+migrating ephemeral resources directory: ephemeral-resources
+migrating file "ephemeral_static.html.markdown"
+extracting YAML frontmatter to "$WORK/templates/ephemeral-resources/ephemeral_static.md.tmpl"
+extracting code examples from "ephemeral_static.html.markdown"
+creating example file "$WORK/examples/ephemeral-resources/ephemeral_static/example_1.tf"
+creating example file "$WORK/examples/ephemeral-resources/ephemeral_static/example_2.tf"
+creating import file "$WORK/examples/ephemeral-resources/ephemeral_static/import_1.sh"
```

Comment:
Not specific to this test, but just a note in general about tests. I see a lot of the internals of resources are already shared with data sources, which I think is fine.

It's a little difficult to see if the expected output below is meant for an ephemeral resource because all of the templates used seem to be geared towards a managed resource (for example, this test has an `import.sh`), just running through the ephemeral resource code path. For this example, it's not immediately clear how to recreate this test archive, although there isn't an existing ephemeral resource to test it on yet.

We could wait until there is, or for some of the generate ones, just take an upcoming ephemeral resource like `random_password` as the example (rather then `null_ephemeral_resource`, which we don't have planned to exist ATM).

Reply:
If we are talking about tests serving as an example of how to use the tool, then I get what you mean. In terms of verifying the functionality, the tests are easy to write because it verifies that the correct files are being created and that all of the template functions are working. But because `terraform-plugin-docs` doesn't care about whether the schema is "valid" or not and to a certain extent what "types" are, the tests end up being very arbitrary. 

We can consider always having at least one real provider test, like the null provider success test, but we don't have a utility provider that implements all schema types now, so we run into the problem of having either an arbitrary provider or using multiple utility providers to cover all types (time and random for example).